mods/cnc/rules/trees.yaml

```diff
@@ -4,6 +4,8 @@ SPLIT2:
 		ResourceType: Tiberium
 		Interval: 55
 	WithIdleAnimation:
+	AppearsOnMapPreview:
```

Comment:
IMO it would be better to keep these next to `RadarColorFromTerrain` on `^TibTree`.